scripts/dts/gen_defines.py

```diff
@@ -68,10 +68,13 @@ def main():
             write_special_props(node)
             write_vanilla_props(node)
 
+        write_child_identifiers(edt)
```

Comment:
I think this should go after the parent print above, to write it for all nodes.

Reply:
Done